wai-aria/role/role_none_conflict_resolution.tentative.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Role None Conflict Resolution Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    
+    <!-- no conflict, since not focusable; inner button should be exposed as generic - children presentational - //https://github.com/web-platform-tests/interop-accessibility/issues/161 -->
+    <h2>ARIA role="button" nesting ARIA role="button" (no presentational roles conflict resolution, since not focusable)</h2>
+    <div role="button">
+      <span>outer button</span>
+      <span role="button" class="ex-generic" data-testname="non-focusable ARIA[role button] nesting non-focusable ARIA[role button]">Inner button</span>
+    </div>
+
+    <!-- conflict arise, since inner button is focusable - //https://github.com/web-platform-tests/interop-accessibility/issues/161-->
+    <h2>HTML button nesting HTML button</h2>
+    <div id="button-container">
+      <button id="outer-button">
+        <span>outer button</span>
+        <span id="inner-button-container"></span>
+      </button>
+    </div>
+
+    <script>
+      AriaUtils.verifyRolesBySelector(".ex");
+      AriaUtils.verifyGenericRolesBySelector(".ex-generic");
+
+      //https://github.com/web-platform-tests/interop-accessibility/issues/161 starts
+      const innerButton = document.createElement("button");
+      innerButton.textContent = " (inner button)";
+      innerButton.id = "inner-button";
+      document.getElementById("inner-button-container").appendChild(innerButton);
+
+      promise_test(async t => {
+        const role = await test_driver.get_computed_role(document.getElementById('inner-button'));
+        assert_equals(role, "button");
+      }, "test nested HTML button within HTML button");
+      //https://github.com/web-platform-tests/interop-accessibility/issues/161 ends
```

Comment:
Don't duplicate this... It's abstracted for you in aria_utils. Just use the data-expected* attributes here like other tests.
```suggestion
```